### DISCUSSION TITLE
How to set the default per page value?

### DISCUSSION THREAD

**Discussion by OvalMedia** [filamentphp/filament#4890] [Help]
It seems like a fundamental task but I could not find anything. How do I set the default value for the number of records displayed per page? Additionally I would be curious about how to configure the entries in the per page selection?

Thank you.

**Comment by danharrin** [comment]
Tables config - https://github.com/filamentphp/filament/blob/2.x/packages/tables/config/tables.php#L45-L57

**Reply by OvalMedia** [reply]
It cannot be set per resource?

**Reply by danharrin** [reply]
On the list page - https://filamentphp.com/docs/2.x/tables/getting-started#pagination

**Reply by OvalMedia** [reply]
I set `public $tableRecordsPerPage = 25;` on my list page but it still defaults to 10.

**Reply by RX-77IT** [reply]
> I set `public $tableRecordsPerPage = 25;` on my list page but it still defaults to 10.

Have you tried using this in the ListModels page?
```php
protected function getTableRecordsPerPageSelectOptions(): array 
{
    return [25, 50, 100];
} 
```
The first value will be the default.
